Makefile

```diff
@@ -516,7 +516,7 @@ install-docker-buildx: ## Create `docker buildx` builder.
 
 
 .PHONY: golangci
-golangci: GOLANGCILINT_VERSION = v1.51.2
+golangci: GOLANGCILINT_VERSION = 1.51.2
```

Comment:
I guess this change in makefile is not intended, as golangci v1.51.2 is a valid one.

Reply:
Yes, you are right, I missed this line when copying code from my own development workspace.